src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -267,7 +267,7 @@ function MoneyRequestConfirmationList({
 
     const currency = (mileageRate as MileageRate)?.currency ?? policyCurrency;
 
-    const distance = transaction?.routes?.route0?.distance ?? 0;
+    const distance = transaction?.comment?.customUnit?.quantity ?? 0;
```

Comment:
Can we reuse [TransactionUtils.getDistance](https://github.com/Expensify/App/blob/35772b2f586aec542c9bf13c222691e623cf8f9b/src/libs/TransactionUtils.ts#L359) and update this logic there?

Reply:
Oh wow thanks I totally missed that. I should have searched for `routes` vs `distance = `. Updating.